src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,57 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final int INDEX_ACTION = 0;
+    private static final int INDEX_AMOUNT = 1;
+    private int buy = 0;
+    private int supply = 0;
```

Comment:
This variables should not be here. They must be local variables in method "createReport"
```suggestion
```